src/components/landing-page/ProblemChooser.tsx

```diff
@@ -57,5 +57,14 @@ export const ProblemChooser = (props: GridProps) => (
         description="Given a list of cities and the distances between them, find the shortest possible route that visits each city exactly once and returns to the origin city."
       />
     </GridItem>
+    <GridItem>
+      <ProblemCard
+        href="solve/Knapsack"
+        new={true}
+        tags={["QUBO"]}
```

Comment:
Please change this to the QAOA tag, because the problem is solved via QAOA.
QUBO is only used when the QUBO problem mask can be used as a subroutine.

Reply:
fixed in commit f6b581fd6e0b2921174d7771f75e84ef8fed4e03